doc/index.md

```diff
@@ -15,6 +15,16 @@ Although the object is in the global scope, it is not available until after the
 You need a Stripe API key for using this plugin, which you can obtain from [Stripe](https://stripe.com/docs). Once you have your API key, you can install the plugin in the following way:
 
     cordova plugin add url —variable API_KEY="YOUR_API_KEY"
+    
+## Security
+
+This plugin is not intended to be used in applications for general client use, as it requires that the Stripe secret key is embedded. As the name suggests, you should never share the secret key as it allows full access to the Stripe API.
```

Comment:
I think I understand what you mean by "general client use", but I'm not sure a casual observer would.

I'd be +1 about some more explicit wording here. It seems to me the bottom line is that "you shouldn't be distributing an app with your embedded secret key to anybody you wouldn't be comfortable making a read/write user on your stripe account". (In particular, a user that you can't decredential without having to update every copy of the app in the wild that has the old secret key.)

An attacker who has gotten a hold of somebody's secret key can do a lot of financial damage to them. I'd err on the side of making this warning more obvious than it needs to be.